### PR TITLE
Jonathangrm belfast list

### PR DESCRIPTION
**Pull request by jonnygrm**
Adding a file to list places to eat in Belfast, Northern Ireland.

:1st_place_medal:

### CODE REVIEW

**Comment by jonnygrm** [comment]
@githubteacher can you review this for me please? Just noticed I listed places to eat outside of Belfast but lets go with this list anyway!

**Comment by jonnygrm** [comment]
Maybe @hectorsector could review this 😸

File: jonathangrm-belfast.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Portstewart, Northern Ireland
+
+## Places to Eat
+- McDonalds not for :beer: :smile:
```
Comment (lumaquan) [inline jonathangrm-belfast.md:4]: At least is cheap :)

**Comment by sadashivkulthe** [comment]
I never been to Belfast before. Can you recommend me places to eat.
I'm pure vegetarian so could you recommend good vegetarian restaurants in your city?

File: jonathangrm-belfast.md (hunk 1 above)
Comment (sadashivkulthe) [reply jonathangrm-belfast.md:4]: I often been to McD... Love Aalo tikki from McD
What do you like from McD

File: jonathangrm-belfast.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- Tides
```
Comment (sadashivkulthe) [inline jonathangrm-belfast.md]: What's special from here?

**Review by sadashivkulthe — COMMENTED** [review]
Can you review changes?

File: jonathangrm-belfast.md (hunk 1 above)
Comment (jonnygrm) [reply jonathangrm-belfast.md:4]: I love their milkshakes :smile:

File: jonathangrm-belfast.md (hunk 2 above)
Comment (jonnygrm) [reply jonathangrm-belfast.md]: I've never been :rofl: Although I hear the food tastes great. It also sits high up, overlooking the beach.

**Comment by jonathangrm** [comment]
This closes #27

**Comment by jonnygrm** [comment]
This closes #15

**Review by hectorsector — APPROVED** [review]
🚢 it!

**Pull request merged**